diff --git "a/C:\\Users\\Darcy.G\\AppData\\Local\\Temp\\TortoiseGit\\buiDE32.tmp\\build-c36986b-left.nmml" "b/D:\\OpenSource\\StablexUI\\samples\\dragDrop_simple\\build.nmml"
index ab5bd70..8b9e6f4 100644
--- "a/C:\\Users\\Darcy.G\\AppData\\Local\\Temp\\TortoiseGit\\buiDE32.tmp\\build-c36986b-left.nmml"
+++ "b/D:\\OpenSource\\StablexUI\\samples\\dragDrop_simple\\build.nmml"
@@ -2,14 +2,12 @@
 <project>
 
     <meta title="StablexUI. Simple drag and drop" package="ru.stablex.ui" version="0.0.1" company="R.U.N" />
-    <app path="./build" file="stablexui" main="com.example.Main"/>
+    <app path="./build" file="stablexui" main="Main"/>
 
     <window width="800" height="600" background="0xFFFFFF" fps="60"/>
 
     <source name="../../src/" />
 
-    <assets path="assets/img" include="*" />
-
     <haxelib name="nme" />
     <haxelib name="actuate" />
 
